package.json

```diff
@@ -187,15 +184,11 @@
     "@bigtest/interactor": "^0.9.3",
     "@bigtest/mocha": "^0.5.1",
     "@bigtest/react": "^0.1.2",
```

Comment:
`Bigtest` tests have been removed in this PR, so `@bigtest` deps are no longer needed.